### PR TITLE
Make $SET_STATUS static

### PR DESCRIPTION
**Pull request by sloan58**
Made $SET_STATUS a private static var and included the '$' access when called in Admin@is_status

### CODE REVIEW

**Comment by mschwager** [comment]
Hey @sloan58! Thanks for the PR! I've added a question inline.

File: src/Admin.php [hunk 1]
```diff
@@ -13,9 +13,9 @@ class Admin extends Client {
      * Values a user/group's status can be set to. Note that this is what
      * they can be SET to, there are additional values that can be retrieved.
      */
-    private $SET_STATUS = array("active", "bypass", "disabled");
+    private static $SET_STATUS = array("active", "bypass", "disabled");
```
Comment (mschwager) [inline src/Admin.php]: What was wrong when it wasn't `static`?

**Comment by mschwager** [comment]
This looks good to me! Can you squash this into a single commit? Then we're good to go :)

**Comment by sloan58** [comment]
I squashed the commits to a single update.  let me know if you need anything else, and thanks for the help!

________________________________

**Comment by mschwager** [comment]
Merged, cheers :)

**Comment by sloan58** [comment]
Thanks!

**Pull request merged**